package.json

```diff
@@ -6,7 +6,7 @@
   "license": "Apache-2.0",
   "homepage": "https://firebase.google.com/",
   "engines": {
-    "node": ">=10.10.0"
+    "node": ">=12.7.0"
```

Comment:
Does Cloud Function still support Node 10? If so, will this prevents people from using `admin-node` with Node 10 in Cloud Function?

Reply:
LGTM, otherwise

Reply:
Their early reaction was that this change is ok. Functions are already defaulting to Node 12 for all new deployments. We will of course bump our major version to indicate this change when we eventually release this.